shell/client/setup-wizard/wizard.html

```diff
@@ -44,7 +44,7 @@
                                    mayJumpTo=(mayJumpTo "identity")
                                    route="setupWizardIdentity"
                                    className=itemClassName}}
-      Login provider(s)
+      Identity providers
```

Comment:
>I think @neynah above says that we have received criticism of the term "identity" being confusing, unless I'm misunderstanding her.

Not exactly! I just got that impression so I wouldn't take it as fact.

Also, I just asked 2 of the users right now and they both prefer "Identity providers"! So I'd say it's pretty safe to go with Drew's choice of string.